axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/EventBuffer.java

```diff
@@ -156,7 +156,12 @@ public boolean hasNextAvailable(int timeout, TimeUnit timeUnit) throws Interrupt
     public TrackedEventMessage<?> nextAvailable() {
         try {
             hasNextAvailable(Integer.MAX_VALUE, TimeUnit.MILLISECONDS);
-            return peekEvent == null ? eventStream.next() : peekEvent;
+            TrackedEventMessage<?> next = peekEvent == null ? eventStream.next() : peekEvent;
+            logger.trace("Polling next available event {} in an Event Buffer {}. Tracking Token {}.",
```

Comment:
"Polled" (or "Consumed") instead of "Polling"? When the log entry is written, the entry is already taken from the buffer.